BUILD-LINUX.md

```diff
@@ -1,93 +1,51 @@
 
 <pre>
 &lt;ROOT&gt;
-   |-libs
-   |   |- sundials
-   |   |- hdf5
-   |   |- superlu
-   |   |- suitesparse
    |-code
    |-cadet
    |-build
 </pre>
 
-Note that the version numbers of the files and packages below are subject to change and will not always reflect the most recent version.
+Note that the version numbers of the files and packages below are subject to change and will not always reflect the most
+recent version.
 
-# Build dependencies
+## Install dependencies
+```
+sudo apt-get update
+sudo apt -y install build-essential libhdf5-dev liblapack-dev libblas-dev libtbb-dev libsuperlu-dev libeigen3-dev
+```
 
-## HDF5
+### LAPACK
 
-You can either build HDF5 yourself or rely on the packages provided by your Linux distribution.
-
-Obtain HDF5 from your distribution:
-* Install the development package of your distribution (e.g., `libhdf5-dev` for Ubuntu and Debian, `hdf5-devel` OpenSUSE)
-* Additional postfixes or prefixes in the library names (e.g., Debian uses `libhdf5_serial.*` instead of `libhdf5.*`) can cause troubles when CMake is trying to find HDF5. If this is the case, you can circumvent this by using symlinks `ln -s libhdf5_serial.* libhdf5.*`. However, this can later cause troubles because other packages might contain this file.
-
-Build HDF5 yourself:
-* Download CMake-enabled source from https://www.hdfgroup.org/downloads/hdf5/source-code/
-* Unzip and make sure that the directory path does not contain blank spaces
-* Open a terminal and change to the unzipped directory
-* Execute `ctest -S HDF5config.cmake,BUILD_GENERATOR=Unix,INSTALLDIR="<ROOT>/Libs/hdf5" -C Release -V`
-* Extract the created `HDF5-1.10.0-patch1-Linux.tar.gz` file to `<ROOT>/Libs/hdf5` such that you have `<ROOT>/Libs/hdf5/lib`
-
-## SUNDIALS (Optional)
-
-* Download SUNDIALS source from http://computation.llnl.gov/projects/sundials/sundials-software (version <= 3.2.1)
-* Unzip
-* Open a terminal and change to the parent directory of the unzipped directory
-* Create a new folder `sundialsbuild` and change to it
-* Execute `cmake -DCMAKE_INSTALL_PREFIX="<ROOT>/Libs/sundials" -DEXAMPLES_ENABLE=OFF -DOPENMP_ENABLE=ON -DBUILD_SHARED_LIBS=OFF -DCMAKE_C_FLAGS=-fPIC ../sundials-3.2.1/`
-* Execute `make install`
-* Delete the folder `sundialsbuild` (e.g., execute `rm -rf sundialsbuild` in the parent directory of `sundialsbuild`)
-
-## LAPACK
-
-You can either use a LAPACK implementation provided by your distribution or install the freely available [Intel MKL](https://software.intel.com/sites/campaigns/nest/) which is very fast and probably faster than your distribution's implementation.
+You can either use a LAPACK implementation provided by your distribution or install the freely available 
+[Intel MKL](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html?operatingsystem=linux&distributions=online) which is very fast and probably faster than your 
+distribution's implementation.
 
 Obtain LAPACK from your distribution:
```

Comment:
The "Install dependencies" section uses the LAPACK implementation "provided by your distribution". This expands that to show how to use Intel MKL.

Reply:
ah ok, I was just confused because it's `liblapack` and `libblas` are already in the dependencies list. Also, did you know, you can use 

```
sudo apt install intel-mkl
```
to install everything?

Reply:
Do you know for sure/ can test in Linux, that the build works if we replace liblapack and libblas with intel-mkl?

Reply:
I've tested this on WSL and can confirm intel-mkl from apt works.